test/templates/popover/index.njk

```diff
@@ -1,13 +1,22 @@
 {% extends "layout.njk" %}
 {% import "./../utils.njk" as utils %}
 {% from "./../format.njk" import format %}
-{% from "./component.njk" import popover, popover_popperjs %}
+{% from "./component.njk" import popover, popover_popperjs, list %}
 {% from "./../menu/component.njk" import menu %}
 
 {% block content %}
 
 <style media="screen">
     .fd-popover{ margin-left: 100px; margin-right: 100px;}
+
+    .popover-body__pgd {
+    font-family: '72', -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
```

Comment:
I would avoid using the font 72 here.  This is a playground page and is only intended to highlight the popover styles -- not the popover content styles.